onnx/test/shape_inference_test.py

```diff
@@ -1915,6 +1915,26 @@ def test_reversesequence(self):  # type: () -> None
             [])
         self._assert_inferred(graph, [make_tensor_value_info('y', TensorProto.FLOAT, (4, 5, 6))])
 
+    def test_unique_without_axis(self):  # type: () -> None
```

Comment:
It'd be nice if you can add more complicated cases (such as 2x3x2 tensor) and some edge cases such as scalar, one-element tensor, etc.